src/lib/constants.js

```diff
@@ -1,8 +1,10 @@
 /* @flow */
 export default {
     'sevenDays': 6.048e+8,
+    'oneMonth': 2592000000,
```

Comment:
nit: might be better if you just did: `1000*60*60*24*30`

Reply:
instead of calculating it and putting the calculated value here :)

Reply:
Ah, yeah maybe we should just get away from the gigantic, cryptic numbers.

Reply:
can we represent it as a roman numeral MMMCMXCIXCMXCIX

Reply:
I'll drop what I'm doing and get started.